backend/tracim_backend/lib/mail_notifier/notifier.py

```diff
@@ -300,7 +299,13 @@ def notify_content_update(self, event_actor_id: int, event_content_id: int) -> N
             # contains only message_id from parents post in thread.
             # To link this email to a content we create a virtual parent
             # in reference who contain the content_id.
-            message["References"] = formataddr(("", reference_addr))
+            references = formataddr(("", reference_addr))
+            # INFO - G.M - 2020-04-03 - Enforce bracket in references header if not provided,
+            # formataddr add them only if label is not empty, but we need them here to ensure best
+            # compat from parsing software
+            if reference_addr[0] != "<":
```

Comment:
Shoud `reference_addr` be checked for emptiness here, to avoid an IndexError?

Reply:
good suggestion.

Reply:
done in EmailAddress.address